ledfx/effects/melt_and_sparkle.py

```diff
@@ -11,7 +11,7 @@
 from ledfx.utils import empty_queue
 
 
-class HuxleyMelt(AudioReactiveEffect, HSVEffect):
+class MeltSparkle(AudioReactiveEffect, HSVEffect):
```

Comment:
Noting I am not overly qualified to review this, but pitching in.

Good to move off personal names +1

Reply:
yeah that was an oversight from my initial hacking :)